VocaDbWeb/Scripts/Components/Discussion/DiscussionFolder.tsx

```diff
@@ -0,0 +1,38 @@
+import Breadcrumb from '@Bootstrap/Breadcrumb';
+import DiscussionIndexStore from '@Stores/Discussion/DiscussionIndexStore';
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useParams } from 'react-router-dom';
+
+import { DiscussionLayout } from './DiscussionRoutes';
+import ViewFolder from './Partials/ViewFolder';
+
+interface DiscussionFolderProps {
+	discussionIndexStore: DiscussionIndexStore;
+}
+
+const DiscussionFolder = observer(
+	({ discussionIndexStore }: DiscussionFolderProps): React.ReactElement => {
+		const { t } = useTranslation(['ViewRes.Discussion']);
+		const { folderId } = useParams();
+
+		React.useEffect(() => {
+			discussionIndexStore.selectFolderById(Number(folderId));

```

Comment:
FIXME: The [`getFolder`](https://github.com/VocaDB/vocadb/blob/future/VocaDbWeb/Scripts/Stores/Discussion/DiscussionIndexStore.ts#L90) function, which is called by `selectFolderById`, must be called only after the [`getFolders`](https://github.com/VocaDB/vocadb/blob/future/VocaDbWeb/Scripts/Stores/Discussion/DiscussionIndexStore.ts#L67) function has been completed, otherwise topics won't show up. See also https://discord.com/channels/309072240639737866/818499255349018624/867402240649723905.

Reply:
Fixed by #965.